lib/rules/no-invalid-role.js

```diff
@@ -112,8 +183,13 @@ module.exports = class NoInvalidRole extends Rule {
 
         const roleValue = AstNodeInfo.elementAttributeValue(node, 'role');
 
-        if (!['presentation', 'none'].includes(roleValue)) {
-          return;
+        if (!VALID_ROLES.includes(roleValue)) {
```

Comment:
I assume roles are not case-sensitive right? So we should check `roleValue.toLowerCase()` right? And make sure to add a test for that.